canonical-web-and-design/ubuntu.com
# Rebuild /openstack page

## Done

- Rebuild page at `/openstack`


## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page against [copy doc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit#). I've suggested a change to the hero text to keep it more inline with the rest of ubuntu.com. 
- The graph mentioned with the slider will be done as a separate [issue](https://github.com/canonical-web-and-design/ubuntu.com/issues/9524) as this will take some time. 

## Issue / Card

Fixes [#3821](https://github.com/canonical-web-and-design/web-squad/issues/3821)
